dora/shell/src/main/java/alluxio/cli/CheckCluster.java

```diff
@@ -0,0 +1,236 @@
+            if (failedWorker.getNetAddress().getRpcPort() == worker.getNetAddress().getRpcPort()) {
+              iterator.remove();
+              break;
+            }
+          }
+        }
+      }
+    }
+    return workerMap;
+  }
+
+  private static void writeFile(FileSystem fs, AlluxioURI testFile, boolean throughOnly)
+      throws ResourceExhaustedException {
+    WriteType writeType = throughOnly ? WriteType.THROUGH : WriteType.CACHE_THROUGH;
+    try (FileOutStream outStream = fs.createFile(testFile,
+        CreateFilePOptions.newBuilder().setWriteType(writeType.toProto()).build())) {
+      outStream.write(TEST_CONTENT.getBytes());
+    } catch (Exception e) {
+      List<BlockWorkerInfo> workers =
+          sPolicy.getPreferredWorkers(sWorkerInfoList, testFile.getPath(), 1);
+      if (!FAILED_WORKER.contains(workers.get(0))) {
+        FAILED_WORKER.add(workers.get(0));
+      }
+    }
+  }
+
+  private static void verifyFileAssignments(FileSystem fs,
+                                            HashMap<BlockWorkerInfo, AlluxioURI> workerMap,
+                                            boolean throughOnly) throws Exception {
+    for (Map.Entry<BlockWorkerInfo, AlluxioURI> entry : workerMap.entrySet()) {
+      BlockWorkerInfo worker = entry.getKey();
+      AlluxioURI testFile = entry.getValue();
+      URIStatus fileInfo = fs.getStatus(testFile);
+      boolean isFailure = false;
+
+      if (throughOnly && fileInfo.getInAlluxioPercentage() != 0) {
+        isFailure = true;
+      } else if (!throughOnly && fileInfo.getInAlluxioPercentage() != 100) {
+        isFailure = true;
+      }
+
+      if (isFailure) {
+        FAILED_WORKER.add(worker);
+      }
+    }
+  }
+
+  private static void verifyFileContent(FileSystem fs, AlluxioURI testFile) throws Exception {
+    try (FileInStream inStream = fs.openFile(testFile)) {
+      byte[] buf = new byte[TEST_CONTENT.length()];
+      int bytesRead = inStream.read(buf);
+      String readContent = new String(buf, 0, bytesRead);
+
+      if (!TEST_CONTENT.equals(readContent)) {
+        throw new Exception("Read content does not match written content!");
+      }
+    }
+  }
+
+  private static void printInfo() {
+    int successfulWorkers = sWorkerInfoList.size();
+    int failedWorkers = FAILED_WORKER.size();
+    int totalWorkers = successfulWorkers + failedWorkers;
+    System.out.println("\nTotal workers: " + totalWorkers);
+    System.out.println(ANSI_GREEN + "Successful workers: " + successfulWorkers + ANSI_RESET);
+    System.out.println(ANSI_RED + "Failed workers: " + failedWorkers + ANSI_RESET);
+
+    if (!FAILED_WORKER.isEmpty()) {
+      System.out.println(ANSI_RED + "\nList of failed workers:" + ANSI_RESET);
+      for (BlockWorkerInfo failedWorker : FAILED_WORKER) {
+        System.out.println("Failed worker capacity and port:");
+        System.out.println(ANSI_RED + "Capacity:" + failedWorker.getCapacityBytes() + " Port:"
```

Comment:
Let's print as the following

<hostname>:<port> Available space <available spce>/<capacity>